docs/SUMMARY.md

```diff
@@ -1,42 +1,40 @@
 # Table of contents
 
-* [Introduction](README.md)
-* [Getting Started](getting-started/README.md)
-
-## User Guide
-
-* [Architecture](user-guide/architecture.md)
-* [Model](user-guide/model.md)
-* [Model Version](user-guide/model_version.md)
-* [Model Endpoint](user-guide/model_endpoint.md)
-* [Model Version Endpoint](user-guide/model_version_endpoint.md)
-* [Model Deployment and Serving](user-guide/model_deployment_serving.md)
-* [Batch Prediction](user-guide/batch_prediction.md)
-* [Transformer](user-guide/transformer.md)
-
-## Examples
+## Introduction
 
-### Deploy Standard Models
-
-* [Deploy PyTorch model](../examples/pytorch/Sample.ipynb)
-* [Deploy SKLearn model](../examples/sklearn/Sample.ipynb)
-* [Deploy Tensorflow model](../examples/tensorflow/Sample.ipynb)
-* [Deploy XGBoost model](../examples/xgboost/Sample.ipynb)
-
-### Deploy PyFunc Model
-
-* [Deploy PyFunc model](../examples/pyfunc/Sample.ipynb)
-* [Deploy PyFunc model with custom Prometheus metrics](../examples/metrics/Sample.ipynb)
+* [Merlin Overview](README.md)
+* [Architecture Overview](getting-started/architecture.md)
```

Comment:
imho, the architecture overview is quite complex, and to avoid overwhelming our users, I prefer to have it as the first entry in the Developer Guides section below.